common-crafting.lic

```diff
@@ -197,6 +197,51 @@ module DRCC
     true
   end
 
+  def get_adjust_tongs(usage, bag, bag_items, belt, adjustable_tongs)
+    
+    DRCC.get_crafting_item('tongs', bag, bag_items, belt) unless DRCI.in_hands?('tongs')
```

Comment:
Will this do what we expect if `usage` is 'shovel'?

Reply:
so, if we want a shovel, and call this:
1. adjustable_tongs is true, tongs are truly adjustable
2. adjustable_tongs is true, tongs are not adjustable
3. adjustable_tongs is false

If (1), gets them,
1. tongs_status nil, tries to adjust, resolves, sends  true or false, if false, stows.
2. tongs_status defined, if shovel, returns true. if tongs, swaps, returns true.

if (2), gets them
1. tongs_status nil, tries to adjust, resolves, sends true or false, if false, stows.
2. tongs_status defined (possible? doubt it), attempts adjust, fails, stows, sends false.

if (3), gets them, returns false. This would be a problem if you wanted a shovel, and weren't using adjustable tongs, and aren't tempering. Perhaps this and the following line (203) should be under the case statement.